.circleci/config.yml

```
@@ -0,0 +1,26 @@
+version: 2
+
+jobs:
+  build:
+    working_directory: ~/identity-style-guide
+    docker:
+      - image: circleci/ruby:2.5.3-node-browsers
+
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install bundle dependencies
+          command: bundle check --path ~/.bundler || bundle install --path ~/.bundler
+      - run:
+          name: Install dependencies
+          command: npm ci
+      - save_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
```

It's a good idea to include a version number in the cache key just in case you need to invalidate the cache for a reason that doesn't include changes to the checksum values. CircleCI also suggests including the architecture of the container in the key.

```suggestion
          key: dependency-cache-v1-{{ arch }}-{{ checksum "package-lock.json" }}-{{ checksum "Gemfile.lock" }}
```

---

Excellent idea! Updating now — thanks for the review! 😎🤘